<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="component1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="component2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="output1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>590</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="output2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>590</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labout1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Output1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labout2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Output2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labmix">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Mixer Output&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Component_1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Component_2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>160</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>360</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="image1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>32</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>image1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>image2</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="original_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="original_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>251</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="S1Label">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>150</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="S2Label">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>350</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="components1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FTMag1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FTPhase1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FTReal1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FTImag1</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="components2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FTMag2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FTPhase2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FTReal2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FTImag2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="outselect">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>output1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>output2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="imgcom1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>img1_c1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>img2_c1</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="imgcom2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>img1_c2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>img2_c2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="mixcomp1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>c1Mag</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c1Phase</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c1Real</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c1Imag</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c1UnitMag</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c1UnitPhase</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="mixcomp2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>420</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>c2Mag</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c2Phase</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c2Real</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c2Imag</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c2UnitMag</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c2UnitPhase</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionload_image1">
   <property name="text">
    <string>load_image1</string>
   </property>
  </action>
  <action name="actionload_image2">
   <property name="text">
    <string>load_image2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>slider1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>S1Label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>855</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>1000</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>S2Label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>855</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>1000</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
